src/app/api/services/dot-dialog/dot-dialog.service.spec.ts

```diff
@@ -14,7 +14,6 @@ const messageServiceMock = new MockDotMessageService({
     'dot.common.dialog.reject': 'No'
```

Comment:
Had to update this because now you can sent accept and reject actions to the alert dialog, I did it to use it in the layout editor when the user select to edit the template or the layout.